roles/kubernetes-apps/ansible/templates/dashboard.yml.j2

```diff
@@ -17,15 +17,11 @@
 #
 # Example usage: kubectl create -f <this_file>
 
-{% if dashboard_namespace != "kube-system" %}
 ---
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: {{ dashboard_namespace }}
-  labels:
-    name: {{ dashboard_namespace }}
-{% endif %}
```

Comment:
I wonder if it would be better to delete the condition?
I tried to test it myself apply namespace will pop up a warning message and leave a record for the annotation.
```
Warning: resource namespaces/kube-system is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
namespace/kube-system configured
```

Reply:
Hum. I wonder what the effects would be when using --prune // --applyset ?
(Some things I'd like to add eventually, even though they are still in alpha currently)

Reply:
Hmmm, I'm not sure. I never use this feature. It was implemented in 1.27 (alpha), but there seems to be no progress at the moment.

Reply:
I prefer to keep the condition, maybe wait for `applyset` to be implemented in beta or GA?
The rest is LGTM. (If you think this is fine, you can give LGTM yourself)

Reply:
You're right, I'll revert that part

Reply:
I thought approver could do it. 😅